sentry-core/src/test/java/io/sentry/core/transport/RetryingThreadPoolExecutorTest.kt

```diff
@@ -157,15 +157,156 @@ class RetryingThreadPoolExecutorTest {
         synchronized(jobBlocker) { jobBlocker.notify() }
         atLeastOneFinished.await()
 
+        // wait for the thread pool to realize that the job indeed finished
+        while (threadPool?.completedTaskCount == 0L) {
```

Comment:
it would be also possible with `kotlinx.coroutines` using `runBlocking`, but it's only al test, never mind.